### PR TITLE
Implement 'interruptible' election setting

### LINKED ISSUE #46: Ending an election doesn't work
As an admin, when I try to end an election prematurely, I get a success message (*), but in the frontend it still shows as active. The voting booth remains open, but it returns the following message once the credentials are entered:
"Currently the election is not open for voting. Try again later."

(*) The success message says it's been _paused_ and it should say _ended_.

 
 

1. Create and setup an interruptible election
2. Submit a couple of votes
3. End the election manually
4. Go to the public view
5. See that the election is displayed as active and the voting booth open
6. Enter some valid credentials and see the message: "Currently the election is not open for voting. Try again later."

**Additional context**
Vocdoni SDK version: v0.0.4-alpha

### PR DESCRIPTION
**Pull request by andreslucena**
This PR introduces a  setting that configures on the Vocdoni API if the election can be paused, canceled or ended before the `end_time`.

If this checkbox is enabled, then we show a "Danger zone" in the admin panel once the Election is in the "vote" step. This allows the admin to:

* Cancel an election: doesn't publish the results
* End an election: publish the results. It's the same result as if the election has reached the configured end_time.
* Pause an election: momentarily doesn't accept votes

Fixes #46

### CODE REVIEW

**Review by microstudi — APPROVED** [review]
Excellent job, worked like a charm!

File: README.md [hunk 1]
```diff
@@ -49,7 +49,13 @@ status are checked every 15 minutes, you can do it with this configuration:
 
 ```crontab
 # Change Elections status on decidim-vocdoni
-0/15 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bin/rails decidim_vocdoni:change_election_status
+0/15 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bin/rails decidim_vocdoni:change_election_status[quiet]
```
[the remark this answers is not in the document]
Comment (andreslucena) [reply README.md]: Yes, much better and cleaner. Done at  6cf9e58de4f3204480285c4e729e9fb021b4737f

File: app/packs/src/decidim/vocdoni/admin/steps/setup_election.js [hunk 2]
```diff
@@ -8,6 +7,28 @@ const setupElectionStep = async () => {
     return;
   }
 
+  const createAnswersValuesOnElection = async () => {
+    const urlPrefix = window.location.pathname.split("/").filter((param) => param !== "steps").join("/");
```
[the remark this answers is not in the document]
Comment (andreslucena) [reply app/packs/src/decidim/vocdoni/admin/steps/setup_election.js]: > Is this causing problems when there's a querystring?

Yes, probably.

> maybe we should consider sending the URL in a `data`attribute?

Done at 51f1c4aa56c6c75dcea5d1a0967679f2a3b0cfda

**Review by microstudi — APPROVED** [review]
excellent!

**Pull request merged**